data/phi0472/phi001/phi0472.phi001.perseus-eng4.xml

```diff
@@ -4,9 +4,9 @@ schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-
      <teiHeader>
           <fileDesc>
                <titleStmt>
-                    <title>Carmina</title>
+                    <title xml:lang="lat">Carmina</title>
                     <title type="sub">Machine readable text</title>
-                    <author n="Catul.">C. Valerius Catullus</author>
+                    <author xml:lang="lat">C. Valerius Catullus</author>
                     <editor role="editor" n="Smithers">Leonard C. Smithers</editor> 
```

Comment:
See my comment on the other file. I have no idea why `<editor role="editor">` was ever a practice.